test/retail_outlet/constants.js

```diff
@@ -20,6 +20,16 @@ const FIXED_PAYMENT_CODE_DETAILS = {
   id: FIXED_PAYMENT_CODE_ID,
 };
 
+const PAYMENTS_BY_FIXED_PAYMENT_CODE_ID = {
+  data: [FIXED_PAYMENT_CODE_DETAILS],
+  has_more: true,
+  links: {
+    href: "https://api.xendit.co/fixed_payment_code/5e0ed8797bc384e60435ec62/payments?limit=1&after_id=61c53c4f6",
```

Comment:
Can we have this uRL replaced with the existing constants that we have? E.g: https://github.com/xendit/xendit-node/blob/master/src/card/authorization.js#L24

Reply:
Done